### PR TITLE
LPS-151907 As a developer using headless endpoints, I'd like to get a country by countryId, a2, a3, countryName, and countryNumber

### PR DESCRIPTION
**Pull request by drewbrokke**
This is an update for:
[LPS-151906](https://issues.liferay.com/browse/LPS-151906)
[LPS-151907](https://issues.liferay.com/browse/LPS-151907)

/cc @pei-jung @erickmonteiroo @ChrisKian @patriciajeanperez @epark18 @david-truong

### CODE REVIEW

**Comment by drewbrokke** [comment]
ci:test:relevant

File: modules/apps/headless/headless-admin-address/headless-admin-address-impl/rest-openapi.yaml [hunk 1]
```diff
@@ -1,7 +1,88 @@
...
+                    type: object
+            required:
+                - regionCode
+                - name
 info:
     description:
         "A Java client JAR is available for use with the group ID 'com.liferay', artifact ID
-        'com.liferay.headless.admin.user.client', and version '1.0.0'."
+        'com.liferay.headless.admin.user.client', and version '1.0.0'.. A Java client JAR is available for use with the
+        group ID 'com.liferay', artifact ID 'com.liferay.headless.admin.address.client', and version '1.0.0'."
```
Comment (pei-jung) [inline modules/apps/headless/headless-admin-address/headless-admin-address-impl/rest-openapi.yaml:85]: Looks like you forgot to remove the first sentence :)

**Review by pei-jung — COMMENTED** [review]
Just one tiny change needed. Everything else looks good to me! 👍

**Comment by drewbrokke** [comment]
ci:test:sf

**Comment by drewbrokke** [comment]
ci:test:relevant

File: modules/apps/headless/headless-admin-address/headless-admin-address-impl/rest-openapi.yaml (hunk 1 above)
Comment (drewbrokke) [reply modules/apps/headless/headless-admin-address/headless-admin-address-impl/rest-openapi.yaml]: Oops! Good catch!

Comment (drewbrokke) [reply modules/apps/headless/headless-admin-address/headless-admin-address-impl/rest-openapi.yaml]: Pushed.

**Review by pei-jung — APPROVED** [review]

**Comment by drewbrokke** [comment]
Pushed fix for regen failure.

**Comment by drewbrokke** [comment]
ci:test:sf

**Comment by drewbrokke** [comment]
ci:test:relevant

**Comment by drewbrokke** [comment]
ci:reevaluate:1327401_3695

**Comment by drewbrokke** [comment]
@epark18 are these failures related?

**Comment by drewbrokke** [comment]
ci:test:relevant

**Comment by drewbrokke** [comment]
@epark18 thank you! If everything else looks good I think we can manually forward.

**Comment by epark18** [comment]
Pushed to BChan [#116723](https://github.com/brianchandotcom/liferay-portal/pull/116723).

**Comment by drewbrokke** [comment]
ci:forward

**Pull request closed without merging**